src/components/TodoApp.tsx

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useState, useContext, useEffect } from 'react';
+import classNames from 'classnames';
+import { FilterStatus } from '../types/FilterStatus';
+import { TodosContext } from '../context/TodosContext';
+import { TodoList } from './TodoList';
+import { TodosFilter } from './TodosFilter';
+import { wait } from '../utils/wait';
+
+export const TodoApp: React.FC = () => {
+  // const [todoTitle, setTodoTitle] = useState('');
+  const [filterStatus, setFilterStatus] = useState(FilterStatus.All);
+
+  /* Unable to add a todo
+    <br />
+    Unable to delete a todo
+    <br />
+    Unable to update a todo
+
+    hide err on request
+  */
+
+  const [todosPrep, setTodos, errorMsg, setErrorMsg] = useContext(TodosContext);
+
+  // left for future tasks so lint doesn't complain
+  if (todosPrep.length === -1) {
```

Comment:
array length cannot be -1
https://stackoverflow.com/questions/32386253/can-the-js-array-length-property-ever-return-a-negative-value